tests/ui_tests/app/page/admin/page.py

```diff
@@ -573,7 +576,7 @@ def create_policy(
         self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.ThirdStep.create_btn)
         self.wait_element_hide(AdminPoliciesLocators.AddPolicyPopup.block)
 
-    def get_all_policies(self) -> [AdminGroupInfo]:
+    def get_all_policies(self) -> [str]:
```

Comment:
Maybe clarify that it's policies' names? Or mention it in docstring

Reply:
added